project_templates/technote_spietex/testn-000/authors.tex

```diff
@@ -1,8 +1,7 @@
 %% DO NOT EDIT THIS FILE. IT IS GENERATED FROM db2authors.py"
 %% Regenerate using:
-%%    python $LSST_TEXMF_DIR/bin/db2authors.py > authors.tex
+%%    python $LSST_TEXMF_DIR/bin/db2authors.py Namespace(mode='spie', noafil=False) 
```

Comment:
Do you know what's going on here with the generated argument to `db2authors.py`? Looks like you're printing a dataclass?

Reply:
oh i had not spoted that - i will check and add the missing .files

Reply:
previously i was not passing arguments -- in the script i print {args} i fixed that